test/core/br_if.wast

```diff
@@ -147,6 +147,12 @@
       )
     )
   )
+
+  (func (export "br_if-multi-values") (param i32) (result i32)
```

Comment:
This test fits better into unwind.wast, which has similar tests for the other branch instructions already. Also, don't name it multi-value since that is confusing with actual multi value semantics.